src/containers/SideNav/SideNav.js

```diff
@@ -115,6 +115,13 @@ export class SideNav extends Component {
             >
               PipelineResources
             </SideNavMenuItem>
+            <SideNavMenuItem
```

Comment:
This should be below the namespace dropdown, sitting alongside the Import Resources link as it's not a Tekton resource